src/index.js

```diff
@@ -302,10 +302,34 @@ const addLang = (tree, {rules: {lang}}) => {
   return tree;
 };
 
+const commentsFormatting = tree => {
+  tree.walk = walk;
+  tree.walk(node => {
+    if (typeof node === 'string' && /<!--[^]*-->/gm.test(node.trim())) {
```

Comment:
1. Regular expression `/<!--[^]*-->/gm` is better to put in a constant
2. The beginning `<!--` and the end `-->` of the notation for the comment I would put into a constant

Reply:
> 2\. The beginning `<!--` and the end `-->` of the notation for the comment I would put into a constant

DOne 👍 

> 1. Regular expression `/<!--[^]*-->/gm` is better to put in a constant

I dont know why but when I am putting that into a constant and using the `.test` method, its not working.

Reply:
you create with `new RegExp` ?

Reply:
I tried, its still the same. I don't know why it's happening. Its working fine for the other regex constants

Reply:
hmm yes that's weird